### PR TITLE
[JOLINT-30] created the feature general grid layout

### PR DESCRIPTION
**Pull request by xxrobone**

### CODE REVIEW

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import GeneralGridLayoutT from './_components/GeneralGridLayout'
```
Comment (ell-ska) [inline app/dashboard/page.tsx:1]: small t at the end, use @ when importing

File: app/dashboard/page.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
 const Dashboard = () => {
-  return <div>dashboard</div>
+  return <div></div>
```
Comment (ell-ska) [inline app/dashboard/page.tsx:4]: you didn't render the grid hahah

**Review by ell-ska — CHANGES_REQUESTED** [review]

File: app/dashboard/page.tsx (hunk 2 above)
Comment (xxrobone) [reply app/dashboard/page.tsx]: @ell-ska updated the general layout import syntax

**Comment by DavidDyberg** [comment]
Test on develop:
Looks good!

**Pull request merged**